src/SwedbankPay.Sdk/SwedbankPayClient.cs

```diff
@@ -13,8 +10,7 @@ namespace SwedbankPay.Sdk
 {
     public class SwedbankPayClient : ISwedbankPayClient
     {
-        public SwedbankPayClient(HttpClient httpClient,
-                                 ILogger logger = null)
+        public SwedbankPayClient(HttpClient httpClient)
```

Comment:
ILogger should be passed so we are able to log errors. With this there is no logging in the SDK

Reply:
The `ILogger` were removed as there were no reference for it.

Reply:
I understand, it was used in SwedbankPayHttpClient to log errors. Check with @asbjornu regarding this.

Reply:
True true. I'll change up the client so it once again logs.

Reply:
We need the logging, but we should also investigate whether the two different client classes can be merged into one.

Reply:
Why do we need the logging?